src/types/real.md

```diff
@@ -1 +1,44 @@
 # Real
+
+`real` is the SML type of real or floating-point numbers. As in other programming languages, reals in SML are restricted to finite machine representations, which means that they cannot represent every real number with perfect precision. For this reason, generally in this course we will prefer the use of `int`s when performing numeric operations.
+
+## Values
+A real number is a sequence of numbers, followed by a decimal point, followed by another sequence of numbers. This includes examples such as `15.150`, `1.0`, and `3.14159`. Reals are noteworthy in that they are not _equality types_, which means that they cannot be compared for equality with the `=` operator. In addition, they cannot be pattern matched upon. This means that when designing programs with specific behavior based on equality with a specific real number, they should instead be written to operate within some _degree of precision_ of the real number in question. For instance:
```

Comment:
Same thing here about equality types - unsure if that's something we should be introducing?